packages/redis/src/RedisProxy.ts

```diff
@@ -25,18 +25,29 @@ export enum RedisEvents {
     End = "end",
 }
 
+interface IRedisStreamOperations {
```

Comment:
do we need this extra interface and type? RedisClient has that function, and since it's already tightly coupled to that type anyways ...

Reply:
nvm, i just read the comment below ... it's probably a workaround for that

Reply:
Yup that is the case